www/%username/identity.spt

```diff
@@ -243,8 +243,9 @@ else:
     % endif
 
     <p>{{ _(
-        "If you give more than {0} in a year, or receive more than {1} in a year, "
-        "or if your account shows signs of suspicious activity, then you will have "
+        "You are required to provide additional information if you give more than {0} in a year, "
```

Comment:
"in" means current calendar year for me, "a year" means within 365 days. Don't know which one it really is. Garden sentence avoided.